website/community/how-to-contribute/bug-reports-feature-requests.md

```diff
@@ -0,0 +1,49 @@
+# Bug reports and feature requests
+
+You can report an issue in the [Fluss issue tracker](https://github.com/alibaba/fluss/issues).
+
+## Issue Reporting Guide
+
+Considerations before opening an issue:
+
+- Make sure that the issue corresponds to a genuine bug or enhancement request. Exceptions are made for typos in documentation files, which can be reported directly.
+
+- Name the issue in the format "[component] Title of the issue", where *[component]* should be replaced by the name of the component being affected. Typically, this corresponds to areas like [server], [client], [connector], [docs]. Skip *[component]* if you are unsure about which is the best component.
```

Comment:
We don't force user to add component name on the issue name, instead, we suggest users to tag the issue with component lables. This makes the issues can be searchable with the lable filter.